src/translations/en.json

```diff
@@ -3,8 +3,10 @@
     "addNewStudyDescription": "To add a new study, you should enter the following information:",
     "caseExist": "Does the case already exist?",
     "studyName": "study name",
+    "studyNameProperty": "study name: ",
     "studyDescription": "study description",
-    "owner": "Owner",
+    "studyDescriptionProperty": "study description: ",
```

Comment:
case-sensitive is is taken into account

Reply:
I agree, added uppercases where necessary.